qdrant-landing/content/documentation/Tutorials & Examples/cloud-inference-hybrid-search.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Using Cloud Inference with Qdrant for Vector Search
+weight: 35
+---
+# Using Cloud Inference with Qdrant for Vector Search
+## Install Qdrant Client
```

Comment:
Could you please describe what we want to achieve in that tutorial before installing the dependencies? It makes sense to start with a sentence about cloud inference and leave room for adding a link to the cloud inference documentation later. Moreover, we built a hybrid search here and used RRF for fusion, so it makes sense to let people know what they can expect, including the problem we are solving.

Reply:
Done